codalab/apps/web/views.py

```diff
@@ -485,6 +485,37 @@ def get_context_data(self, **kwargs):
         context['site'] = Site.objects.get_current()
         context['current_server_time'] = datetime.now()
 
+        # Top 3 Leaderboard
+        # Get the month from submitted_at
+        try:
+            truncate_date = connection.ops.date_trunc_sql('day', 'submitted_at')
+            score_def = SubmissionScoreDef.objects.get(competition=competition, ordering=1)
+            qs = SubmissionScore.objects.filter(result__phase__competition=competition, scoredef__ordering=1)
+            qs = qs.extra({'day': truncate_date}).values('day')
+            if score_def.sorting == 'asc':
+                best_value = Max('value')
+            else:
+                best_value = Min('value')
+            qs = qs.annotate(high_score=best_value, count=Count('pk'))
+            context['graph'] = {
+                'days': [s['day'].strftime('%d %B %Y')  # ex 24 May 2017
+                       for s in qs],
+                'high_scores': [s['high_score'] for s in qs],
+                'counts': [s['count'] for s in qs],
+                'sorting': score_def.sorting,
+            }
+        except:
```

Comment:
You *almost* never want to do a `try ... except` that catches all exceptions.

Let's screenhero and figure this out!